src/modules/network/implementation/libp2p-service.js

```diff
@@ -222,6 +220,7 @@ class Libp2pService {
                     message.header.keywordUuid,
```

Comment:
Add handling when valid is false or busy is true. Add try/catch block with just logging